.github/workflows/build_playground_preview.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy VRL Playground Preview Site
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  deploy_preview_site:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+
+    # Create the HMAC key
+    - name: Start Amplify Build
+      id: hmac_runner
+      env:
+        REQUEST_TOKEN: ${{ secrets.REQUEST_TOKEN }}
+        REQUEST_MESSAGE: ${{ secrets.REQUEST_MESSAGE }}
+        ENDPOINT: ${{ secrets.BUILDER_ENDPOINT }}
+      run: |
+        HMAC_KEY=$(echo -n $REQUEST_MESSAGE | openssl dgst -sha256 -hmac "$REQUEST_TOKEN" -binary | od -An -tx1 | tr -d ' \n'; echo)
+        SIGNATURE="sha256=$HMAC_KEY"
+
+        RESPONSE_CODE=$(curl -s -o /dev/null -w "%{http_code}" -X POST \
+             -H "Content-Type: application/json" \
+             -H "X-Hub-Signature: $SIGNATURE" \
+             -d "{\"app_id\": \"d2lr4eds605rpz\", \"branch_name\": \"${{ github.head_ref }}\"}" \
```

Comment:
Nit: we could pull up the `app_id` into a workflow environment variable since we refer to it twice.